sdk/containerservice/Azure.ResourceManager.ContainerService/CHANGELOG.md

```diff
@@ -1,15 +1,22 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0 (2022-09-26)
 
-### Features Added
+This is the first stable release of the Container Service Management library.
 
 ### Breaking Changes
 
+- Changed the type of property `ClientId` in `ManagedClusterServicePrincipalProfile` from `string` to `Guid`.
```

Comment:
The change is the other direction, changed from `Guid` back to `string`.

Reply:
My bad, I missed ==